Sources/TransferMethod/ListTransferMethodPresenter.swift

```diff
@@ -67,7 +67,7 @@ final class ListTransferMethodPresenter {
             switch transferMethodType {
             case "BANK_ACCOUNT":
                 deactivateBankAccount(token)
-            case "BANK_CARD":
+            case "BANK_CARD", "WIRE_ACCOUNT":
```

Comment:
"Wire_ACCOUNT" should be in the "BANK_ACCOUNT" case not BANK_CARD

Reply:
Thanks, changed